src/DynamicMvvm.Abstractions/MvvmConfiguration.cs

```diff
@@ -17,12 +17,12 @@ public static class DynamicMvvmConfiguration
 		/// </summary>
 		public static ILoggerFactory LoggerFactory { get; set; } = new NullLoggerFactory();
 
-		public static ILogger<T> Log<T>(this T _)
```

Comment:
Don't change this. It's a breaking change.

Reply:
This was an old change that was reverted in the last commit

Reply:
It was put to public in [this commit](https://github.com/nventive/Chinook.DynamicMvvm/pull/71/commits/5fd6cd46aa1125acaea2e21618c4d2726db5d3f2)